### PR TITLE
Adds examine text for several quirks with Empath.

### PR DESCRIPTION
**Pull request by Shoddd**
## About The Pull Request

Adds several new examine texts to empath which show social or outward facing quirks.

## Why It's Good For The Game

Empath is currently a 8 point quirk that doesn't have very many upsides. This pr gives it more reasons to be taken as a 8 point quirk and also aims to have negative quirks be seen by others more for good or bad.

## Changelog

:cl:
balance: empath shows some quirks on examine
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -306,6 +306,20 @@
...
+				if (HAS_TRAIT(src, TRAIT_PACIFISM))
+					msg += "[t_He] doesn't look like [t_He] would hurt a fly.\n"
+				if (HAS_TRAIT(src, TRAIT_CLUMSY))
+					msg += "[t_He] looks like a clutz.\n"
+				if (HAS_TRAIT(src, TRAIT_GRABWEAKNESS))
+					msg += "[t_He] seems like a pushover.\n"
+				if (HAS_TRAIT(src, TRAIT_SKITTISH))
+					msg += "[t_He] is a bit skittish.\n"
+				if (HAS_TRAIT(src, TRAIT_ANXIOUS))
+					msg += "[t_He] looks awkward.\n"
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/examine.dm:318]: "awkward" seems a bit, well, awkwardly used here. would "anxious" work better?

Comment (Shoddd) [reply code/modules/mob/living/carbon/human/examine.dm:318]: how does "nervous" sound was thinking about that instead originally.

Comment (PapaMichae1) [reply code/modules/mob/living/carbon/human/examine.dm:318]: Something that leans into the "social" part of it being social anxiety perhaps? Like "seems uncomfortable around people" or even "has trouble making eye contact"?

Comment (Shoddd) [reply code/modules/mob/living/carbon/human/examine.dm:318]: I do like that but feels long, does "seems uncomfortable" sound right?

Comment (Shoddd) [reply code/modules/mob/living/carbon/human/examine.dm:318]: genders are handled automatically [t_He] is she, he, it depending on their gender so best to imagine any "theys" as that.

**Review by Neocloudy — COMMENTED** [review]
examine.dm has stuff for seems/seem [`p_s()`], and also is/are [`t_is`] and it looks like you forgot to use those

case is also relevant and this would capitalize a pronoun in the middle of a sentence

**Comment by ExcessiveUseOfCobblestone** [comment]
I like the change but the laundry list check of all these traits is starting to look a bit scuffed

File: code/modules/mob/living/carbon/human/examine.dm (hunk 1 above)
Comment (Shoddd) [reply code/modules/mob/living/carbon/human/examine.dm]: I forgot to consider all pronouns in original wording, reworded all descriptions and changed "awkward" to "nervous" along with a few other things, will mark this as resolved for now.

**Comment by Shoddd** [comment]
The list is getting a bit long, but I don't think it will get longer. I will look for a way to make it more compressed if possible (still learning) but worried that would hurt its readability.

**Comment by ExcessiveUseOfCobblestone** [comment]
was thinking maybe something like

```DM
var/list/static/trait_lister = list(
                                     TRAIT_PACIFISM = "[t_He] do[p_es()]n't look like [p_they()] would hurt a fly.\n",
                                     TRAIT_BLAH = "blahblah.\n",
                                    )
for(var/the_trait in trait_lister)
    if(HAS_TRAIT(src, the_trait))
        msg += trait_lister[the_trait]

**Review by Neocloudy — COMMENTED** [review]
oops left this here but yeah spelling stuff

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -306,6 +306,20 @@
...
+				if (HAS_TRAIT(src, TRAIT_PACIFISM))
+					msg += "[t_He] do[p_es()]n't look like [p_they()] would hurt a fly.\n"
+				if (HAS_TRAIT(src, TRAIT_CLUMSY))
+					msg += "[t_He] look[p_s()] like a clutz.\n"
+				if (HAS_TRAIT(src, TRAIT_GRABWEAKNESS))
+					msg += "[t_He] seem[p_s()] like a pushover.\n"
+				if (HAS_TRAIT(src, TRAIT_SKITTISH))
+					msg += "[t_He] [t_is] a bit skittish.\n"
+				if (HAS_TRAIT(src, TRAIT_ANXIOUS))
+					msg += "[t_He] [t_has] nervous look.\n"
```
[the remark this answers is not in the document]
Comment (Shoddd) [reply code/modules/mob/living/carbon/human/examine.dm]: thanks, fixed, I missed a lot of grammar stuff.

**Comment by Shoddd** [comment]
I'm not really sure on this, I suppose it does look nicer but I feel as if it makes it harder to read, more complex, and longer for no other reason than looking less scuffed. I assume by scuff you just mean how its the same thing on repeat and not fancy looking almost? do you think its necessary? If so I'll make the change and figure it out.
@ExcessiveUseOfCobblestone

**Comment by optimumtact** [comment]
bloat change

**Comment by Shoddd** [comment]
are you referring to the pr or change in formatting by cobble, confused.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 3]
```diff
@@ -306,6 +306,20 @@
...
+				if (HAS_TRAIT(src, TRAIT_GRABWEAKNESS))
+					msg += "[t_He] seem[p_s()] like a pushover.\n"
+				if (HAS_TRAIT(src, TRAIT_SKITTISH))
+					msg += "[t_He] [t_is] a bit skittish.\n"
+				if (HAS_TRAIT(src, TRAIT_ANXIOUS))
+					msg += "[t_He] [t_has] a nervous look.\n"
+				if (HAS_TRAIT(src, TRAIT_SOFTSPOKEN))
+					msg += "[t_He] seem[p_s()] shy.\n"
+				if (HAS_TRAIT(src, TRAIT_FRIENDLY))
+					msg += "[t_He] [t_has] a friendly look.\n"
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/human/examine.dm:322]: All these newlines are going to make this already long examine block even longer. I'd prefer if the quirks were listed in a single line, like this:
```
var/list/empath_quirks_message = list()
if (HAS_TRAIT(src, TRAIT_PACIFISM))
	empath_quirks_message += "peaceful"
if (HAS_TRAIT(src, TRAIT_CLUMSY))
	empath_quirks_message += "klutzy"
. . . . 
msg += "[t_He] also seem[p_s()] quite [english_list(empath_quirks_message)].\n"
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/examine.dm (hunk 3 above)
Comment (Ghommie) [reply code/modules/mob/living/carbon/human/examine.dm:322]: You could also do a mix of mine and Cobbie's requests. I'm not a huge fan of long columns of `if` statements.

**Comment by Shoddd** [comment]
alright thanks for the tips, Im gonna close this pr for now since I need some time to work stuff out and don't know when I'll work on it. May reopen it sometime later, if anyone sees this though and wants to do it themselves feel free to.

**Pull request closed without merging**